### PR TITLE
Support for multi-node signer nets via tupelo's docker-compose stack

### PR DESCRIPTION
**Pull request by cap10morgan**
[Trello card](https://trello.com/c/QNX3nMf2)

This allows specifying `docker-compose: true` under a tupelo config that uses the current directory's `docker-compose.yml` file to spin up that tupelo and run the configured testers against that (either via direct client connection in the case of the Go SDK but via RPC proxy for other SDKs).

### CODE REVIEW

**Comment by cap10morgan** [comment]
Example usage: https://github.com/quorumcontrol/tupelo/pull/289

File: main.go [hunk 1]
```diff
@@ -193,10 +346,14 @@ func run(cfg *config) {
...
-	Name    string   `yaml:"name"`
-	Build   string   `yaml:"build"`
-	Image   string   `yaml:"image"`
-	Command []string `yaml:"command"`
+	Name          string            `yaml:"name"`
+	Build         string            `yaml:"build"`
+	Image         string            `yaml:"image"`
+	Command       []string          `yaml:"command"`
+	Env           map[string]string `yaml:"env"`
+	DockerCompose bool              `yaml:"docker-compose"`
```
Comment (brandonwestcott) [inline main.go]: Might be yagni, but I think docker-compose being a string for the filename might be useful?

Comment (cap10morgan) [reply main.go]: Yeah, and I _almost_ started there, but then thought "YAGNI" and held back. :)

I promise to add it the first time we need it, deal?

Comment (brandonwestcott) [reply main.go]: yagni ftw 👍

**Pull request merged**